datasafe-rest-impl/src/main/java/de/adorsys/datasafe/rest/impl/exceptions/RestImplException.java

```diff
@@ -0,0 +1,8 @@
+package de.adorsys.datasafe.rest.impl.exceptions;
+
+public class RestImplException extends RuntimeException {
```

Comment:
Better to have 2 separate exceptions - UserDoesNotExists,UserExistsException

Reply:
ok.